features/google/src/locales/en/infoScreens/commonInfoScreen.json

```diff
@@ -3,7 +3,7 @@
     "baseInfo.title2": "About the data",
     "bubble.chart": "This diagram is called a <strong>bubble chart</strong>. Each bubble represents one datatype. The number of bubbles shows how many data types there are. The size of each bubble is dependent on the amount of data inside.",
     "bar.chart": "This diagram is called a <strong>bar chart</strong>. The length of each bar is proportional to the values that it represents.",
-    "infographic.value.category": "(Value) Category",   
+    "infographic.value.category": "(value) category",   
```

Comment:
Is that related?

Reply:
Well, I've noticed that the info screens translations are not quite right when I wanted to close PROD4POD-1634. Also, I don't know why `en` is lowercase while `de` is uppercase, but now they should be as in the ticket.